### DISCUSSION TITLE
Allow configure of excel export file location

### DISCUSSION THREAD

**Discussion by g24swint** [dbeaver/dbeaver#15521] [Q&A]
**Is your feature request related to a problem? Please describe.**
When usung the "export to excel" button in the UI (not the query export wizard), the excel file is dropped in a local temp space. This means that excel will not recognize it in the recently used files list.

**Describe the solution you'd like**
I'd like to be able to set the default temp file location for excel file exports to be a directory of my choosing.

**Describe alternatives you've considered**

**Additional context**

**Comment by ShadelessFox** [comment]
Hello @g24swint,

The purpose of this button is to provide a fast way to preview your data in excel.

You can use the data transfer to perform export in .xlsx format and specify the destination folder and file name.

You can find instructions here: https://dbeaver.com/docs/wiki/Data-transfer/.
